pom.xml

```diff
@@ -1742,6 +1745,17 @@
                 </exclusions>
             </dependency>
 
+            <!-- Kubernetes/Operator dependencies -->
+            <dependency>
+                <groupId>io.fabric8</groupId>
+                <artifactId>crd-generator-api</artifactId>
+                <version>${kubernetes-client.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.fabric8</groupId>
+                <artifactId>crd-generator-apt</artifactId>
+                <version>${kubernetes-client.version}</version>
+            </dependency>
```

Comment:
I believe this should go to the `operator` module as well for now – until we have Quarkus/Fabric8 versions more aligned across the project.

Reply:
done